kubernetes/doc/create-options.md

```diff
@@ -0,0 +1,146 @@
+# Kubernetes createOptions Extensions
+
+IoT Edge on Kubernetes supports several experimental features to allow a user to assign additional parameters to pods that represents IoT Edge Modules.
+
+## Experimental features
+
+All features described in this document are experimental. To enable them, the following environment variable need to be set for the edgeAgent (make note of the double underscores):
+
+| Environment Variable Name                  | value  |
+|--------------------------------------------|--------|
+| `ExperimentalFeatures__Enabled`            | `true` |
+
+## Create Options
+
+We added CreateOptions for experimental features on Kubernetes. These options will be ignores until both `ExperimentalFeatures__Enabled` and `ExperimentalFeatures__Enabled{FeatureName}` are not set to `true`.
```

Comment:
`ExperimentalFeatures__{FeatureName}` - remove the extra "Enabled"

Reply:
Do we need to have each extension enabled?  Does it make more sense to have a "k8sextensions"?

Reply:
That what I thought initially. But @veyalla suggested flag per section instead.